java/core/src/java/org/apache/orc/OrcConf.java

```diff
@@ -37,7 +37,7 @@
   ROW_INDEX_STRIDE("orc.row.index.stride",
       "hive.exec.orc.default.row.index.stride", 10000,
       "Define the default ORC index stride in number of rows. (Stride is the\n"+
-          " number of rows n index entry represents.)"),
+          " number of rows an index entry represents.)"),
```

Comment:
Let's avoid touching irrelevant part. You can fix this as a separate PR.

Reply:
I don't think we need a separate jira/pr for this. I'll commit it separately.